packages/frontend-shared/windi.config.ts

```diff
@@ -23,6 +23,7 @@ export const defaultConfig: WindiCssOptions = {
     },
     cursor: {
       'ew-resize': 'ew-resize',
+      'pointer': 'pointer',
```

Comment:
Can you comment on why this change is necessary?
I would feel that this class exists by default.

Reply:
Found the issue, `cursor: { ... }` should be under the `extend` property. Since it wasn't, it was removing all of the default cursor properties. Moving it under `extends` fixes this.